app/models/sales_log.rb

```diff
@@ -42,7 +42,7 @@ class SalesLog < Log
   }
   scope :filter_by_organisation, ->(org, _user = nil) { where(owning_organisation: org) }
 
-  OPTIONAL_FIELDS = %w[saledate_check purchid monthly_charges_value_check old_persons_shared_ownership_value_check othtype discounted_sale_value_check].freeze
+  OPTIONAL_FIELDS = %w[saledate_check purchid monthly_charges_value_check old_persons_shared_ownership_value_check othtype discounted_sale_value_check buyer_livein_value_check].freeze
```

Comment:
are all soft validation interruption screens optional fields? If so is there a way of doing this dynamically since they all end in _value_check?
I'm guessing the answer to the first question is no, otherwise this list would be quite a bit longer!

Reply:
Not all, only the ones that appear in setup section, because if they weren't optional the log would clear every time the validation triggers as all setup fields need to be completed before the rest of the sections are unlocked

Reply:
makes sense thanks

Reply:
although I still wonder if it might not be nicer to amend the `setup_completed?` method rather than keep a static list here if all the pages we ignore end in _check

Reply:
Does the rule to end it in `_check` still exists?